test/lib/index-standalone-no-baseline.ts

```diff
@@ -131,7 +131,7 @@ describe('Test End 2 End - Standalone mode without baseline', () => {
         }
       });
 
-    const result = await getDelta();
```

Comment:
we don't use result anymore?

Reply:
we assign the variable but don't use it, we do still need to call `getDelta()`